src/components/controls/Collapse/Collapse.module.scss

```diff
@@ -12,4 +12,13 @@
     text-align: left;
     width: 100%;
   }
+
+  // Focusable elements inside a collapsed section must be hidden
+  // https://dequeuniversity.com/rules/axe/4.1/aria-hidden-focus
+  :not(span)[aria-hidden='true'] {
```

Comment:
Do these do a jumpy disappear?
Could we use content visibility on the whole content and transition it like in ADS?

Reply:
Hmm, yeah it is a bit jumpy :/ 
I'll see if I can do that instead!